src/octopus-cli.ts

```diff
@@ -7,17 +7,16 @@ import {
   extractZip
 } from '@actions/tool-cache'
 import {debug, info, setFailed} from '@actions/core'
-import {Downloads} from './download'
 import {HttpClient} from '@actions/http-client'
 import {join, dirname} from 'path'
 import {v4} from 'uuid'
+import {OctopusCLIVersionFetcher} from './octopusCLIVersionFetcher'
 
 const osPlatform: string = os.platform()
-const platform: string =
-  osPlatform === 'win32' ? 'win' : osPlatform === 'darwin' ? 'osx' : 'linux'
 const ext: string = osPlatform === 'win32' ? 'zip' : 'tar.gz'
-const octopusTools = `https://download.octopusdeploy.com/octopus-tools`
-const latestUrl = `${octopusTools}/latest.json`
+const baseUrl = `https://g.octopushq.com/`
```

Comment:
we've got a new short-link service (using short.io) under the https://oc.to domain (rather than g.octopushq.com) that we should start using.

Reply:
Added #298 for this feature once it's live